spec/models/invenio_rdm_record_converter_spec.rb

```diff
@@ -816,4 +816,35 @@
       end
     end
   end
+
+  describe "#map_collections_to_communities" do
+    let(:map_collections_to_communites_irrc) { described_class.new(generic_file, collection_store.data) }
+    let(:expected_invenio_ids) { ["biostatistics-collaboration-center-lecture-series", "center-for-community-health"] }
+
+    context "there is not a match in the collection store to the communities mapping json" do
+      before do
+        # clear the existing collections
```

Comment:
I figured out my issue from the previous iteration. I had declared a `before` block such that it would run before each `context` block. Database cleaner _was_ running in between each example, but then the before block was running after to re-insert the collections. Using two before blocks fixes the issue.